applesoft.s

```diff
@@ -436,6 +436,65 @@ WGAmpersand_CHKBX:
 
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; WGAmpersand_PROG
+; Create a progress bar
+; &PROG(id,x,y,width)
+WGAmpersand_PROG:
+	jsr WGAmpersandBeginArguments
+
+	jsr WGAmpersandIntArgument
+	sta WGAmpersandCommandBuffer+0
+	jsr WGAmpersandNextArgument
+
+	jsr WGAmpersandIntArgument
+	sta WGAmpersandCommandBuffer+1
+	jsr WGAmpersandNextArgument
+
+	jsr WGAmpersandIntArgument
+	sta WGAmpersandCommandBuffer+2
+	jsr WGAmpersandNextArgument
+
+	jsr WGAmpersandIntArgument
+	sta WGAmpersandCommandBuffer+3
+
+	jsr WGAmpersandEndArguments
+
+	CALL16 WGCreateProgress,WGAmpersandCommandBuffer
+
+	LDY_ACTIVEVIEW				; Flag this as an Applesoft-created view
+	lda #VIEW_STYLE_APPLESOFT
+	ora WG_VIEWRECORDS+4,y
+	sta WG_VIEWRECORDS+4,y
+	
+	jsr WGPaintView
+	jsr WGBottomCursor
+
+	rts
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; WGAmpersand_SETV
+; Set value (state)
```

Comment:
Let's make this more descriptive-  perhaps "Set the 'value' field of the selected view"